src/fu-util-common.c

```diff
@@ -557,6 +557,12 @@ fu_util_release_get_name (FwupdRelease *release)
 			 * the first %s is the device name, e.g. 'Intel ThunderBolt` */
 			return g_strdup_printf (_("%s Controller Update"), name);
 		}
+		if (g_strcmp0 (cat, "X-ThunderboltController") == 0) {
+			/* TRANSLATORS: the Thunderbolt controller is a device that
+			 * has other high speed Thunderbolt devices plugged into it;
+			 * the first %s is the system name, e.g. 'ThinkPad P50` */
```

Comment:
I don't think this comment is strictly correct.  Thunderbolt controllers are in all sorts of devices, not just host controllers in systems.  Especially once we get into the USB4 timeframe they'll be quite ubiquitous.

Not sure how important that will be for translators though.

Reply:
I figured when we have USB 4 we'd just call them a "USB 4 Controller" or something.

Reply:
I guess we can certainly switch things around based on what happens then.